src/stylesheet.css

```diff
@@ -0,0 +1,89 @@
+div{
```

Comment:
The semantics of a div are display:block .. this could lead to confusion here. Better use a class to specify inline block

Reply:
yeah while editing I got confused myself and couldn't tell what was what anymore